### DISCUSSION TITLE
Future Directions of the project

### DISCUSSION THREAD

**Discussion by MDeLuise** [MDeLuise/plant-it#147] [Announcements]
Hi Plant-it community!
I'm excited to share some recent milestones we've achieved and discuss the plans for the future of the project.

## Recent Achievements
In the latest release, a brand-new web app was introduced. After acknowledging the challenges with the previous version in terms of functionality and user experience, I decided to take the time to rewrite it entirely. The new web app is now built with Flutter and is currently accessible directly from the server container. However, one of the goals is to make it available on app stores, starting with Android, in the near future. This will provide users with the option to continue using the web app or download the mobile app for a smoother experience.

## Future Directions
While Plant-it initially began as a companion app for gardening, the growing community around it has inspired me to explore additional features, particularly social elements. I envision incorporating functionalities for sharing plant care tips, photos, and useful links among users. This means introducing community interaction within the project, allowing users to share information within the same instance (e.g. shared plants with partners) and across different instances. I'm particularly drawn to the idea of creating a fediverse around Plant-it, akin to the federated social networks, where gardening enthusiasts can connect, converse, and share experiences.
Implementing this vision may pose technical challenges, especially concerning the integration of the ActivityPub protocol for federated social networking. However, I'm committed to exploring this possibility and seeing if we can make it a reality. 

Additionally, I plan to incorporate the Trefle dataset into the project, possibly by deploying a dedicated container. This integration would eliminate the need for users to create Trefle accounts, depending on the availability of the service.

## Immediate Focus
Before diving into the complexities of the fediverse and ActivityPub, immediate focus is on ensuring stability and making improvements. Despite the progress with the new web app, there are still gaps, particularly regarding desktop support. We also need to implement thorough testing, especially on the frontend, to catch any bugs and enhance user experience. Therefore, immediate goals include:
- Fixing any bugs that may arise with the new app
- Enhancing support in the GitHub issue section to address community feedback promptly
- Investigating the process of publishing the app on various app stores for broader accessibility

I'm excited about the journey ahead and grateful for your continued support and feedback to the project! 🪴
Feel free to share your thoughts and suggestions in the comments.

**Comment by KickingAss2024** [comment]
I nice job on this open source project, keep up the work. Just one thing to keep the app as an open-source would suggest F-Droid as it offers free and open-source.

Also is beta version to available to help. For the Android app
